### ISSUE TITLE
Are there any details about NT_Xent?

### ISSUE THREAD

**Issue by 980202006** [jhtonyKoo/e2e_music_remastering_system#5]
I wrote the training code myself, but the model did not converge, the loss is always around 5.2, and the loss is how much the model will converge?
Thx.

**Comment by jhtonyKoo** [comment]
Dear user, I'm so sorry that I'm replying to this issue with a late notice.

Please find the below image with the training loss curve of the model.
![MEE_loss_curve](https://user-images.githubusercontent.com/98936546/198507033-d6b1f9e0-cada-474f-9e24-2b7319a9d837.png)

The loss value should be drastically reduced at the beginning and slowly be converged after around 100 epochs of training. Note that since the NX_Xent loss value differs according to the batch size (this model had a batch size of 380 where we used 10 NVIDIA GeForce RTX 3090 GPUs), the loss value might differ from ours.

Please feel free to ask any further questions.
